studies/models.py

```diff
@@ -72,6 +76,32 @@ class Meta:
             ('can_view_demographics', 'Can View Demographics'),
         )
 
+    @property
+    def begin_date(self):
+        active_logs = self.logs.filter(action='active')
+        if active_logs:
```

Comment:
I could be misremembering but I think querysets are always truthy. I think you want to do `if active_logs.exists():`